sansa-owl-spark/src/main/scala/net/sansa_stack/owl/spark/stats/OWLStats.scala

```diff
@@ -0,0 +1,511 @@
+package net.sansa_stack.owl.spark.stats
+
+import org.apache.jena.vocabulary.XSD
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.semanticweb.owlapi.model._
+
+import net.sansa_stack.owl.spark.extractOWLAxioms
+import net.sansa_stack.owl.spark.rdd.FunctionalSyntaxOWLAxiomsRDDBuilder
+
+/**
+  * A Distributed implementation of OWL Statistics.
+  *
+  * @author Heba Mohamed
+  */
+class OWLStats(spark: SparkSession) extends Serializable {
```

Comment:
Some methods start with a capital letter, some don't.

Reply:
Many doc strings were seemingly copied from the RDF stats version but not adjusted accordingly (e.g. triples vs. axioms)

Reply:
revised and corrected.